clients/go/admin/config.go

```diff
@@ -68,6 +67,11 @@ type Config struct {
 	PkceConfig pkce.Config `json:"pkceConfig" pflag:",Config for Pkce authentication flow."`
 
 	Command []string `json:"command" pflag:",Command for external authentication token generation"`
+
+	// Set the ServiceConfig formatted as a json string https://github.com/grpc/grpc/blob/master/doc/service_config.md
+	// eg. { "loadBalancingConfig": [{ "round_robin":{} }], "methodConfig": [{ "name": [{ "service": "foo", "method": "bar" }, { "service": "baz" }], "timeout": "1.000000001s"}]}
+	// find the full schema here https://github.com/grpc/grpc-proto/blob/master/grpc/service_config/service_config.proto#L625
+	ServiceConfig string `json:"serviceConfig" pdflag:",Set the service config for the grpc client"`
```

Comment:
Maybe `DefaultServiceConfig` reflects it better because it is the default if a service doesn't enforce override. ;)